styles.css

```diff
@@ -922,3 +956,220 @@ div#sidenav-line {
   border-bottom: 5px solid black;
   border-radius: 10px;
 }
+
+.popup-image {
+  background-image: url(./Assets/Popup/popup-mobile.png);
+  background-repeat: no-repeat;
+  background-size: 20%;
+  background-position: 50% 50%;
+}
+
+.popup-technologies {
+  list-style-type: none;
+}
+
+a.button {
+  display: flex;
+  display: inline-flex;
+  color: white;
+  border: none;
+  border-radius: 3%;
+  cursor: pointer;
+  font-size: 14px;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+a.popup-button {
+  color: white;
+  text-decoration: none;
+}
+
+.desktop-container {
+  color: red;
+  width: 57.5rem;
+  height: 67.6875rem;
+}
+
+.modal {
+  margin: 4vw;
+  width: 80vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  gap: 0.4rem;
+  padding: 1.3rem;
+  min-height: 250px;
+  position: absolute;
+  background-color: white;
+  border: 1px solid #ddd;
+  border-radius: 15px;
+  z-index: 2;
+}
```

Comment:
- [ ] Kindly make sure the design of the popup modal follows the given Figma design as I noticed there are some issues in your design as outlined below 👇
- Your design:
  ![image](https://github.com/dvcorreia1984/portfolio/assets/104968236/3fbde98b-42cb-4bfe-8670-e736b8b891ee)
![firefox_AQPzMJocDm](https://github.com/dvcorreia1984/portfolio/assets/104968236/23d9db9a-2cc8-4ada-ac0d-ee90024bb7e7)
- Figma design:
  ![image](https://github.com/dvcorreia1984/portfolio/assets/104968236/7811a670-73f7-4668-98e2-18b3de22dc43)
- [ ] I've mentioned some of the issues in the screenshot but I highly advise you to pay attention to the overall design (like paragraph color...) as a frontend developer you have to master such skills so you have a better chance of impressing future clients.